### ISSUE TITLE
AG Grid React with ReactUI and left pinned column

### ISSUE THREAD

**Issue by alippai** [ag-grid/ag-grid#4804]
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[] support request => see 'Requesting Community Support'
```

**Current behavior**
When I have a left pinned column and turn the react UI on, the first column values disappears (the header is still shown). Checking the DOM it looks like the left pinned column is zero width, the rest of the columns take up all the space.

When I resize the column (setting the size) the values reappear.
If I turn off either the pinning or the react UI, the column values are shown.
The rowgroup (left pinning) is missing the width (min-width, max-width) css values initially, but after resizing they are computed. Similarly if I turn off the pinning or the react UI they are computed even initially.

**Expected behavior**
Values are shown in every case

* **AG Grid version:** 26.2.0

* **Browser:**
Chrome

* **Language:** [ TypeScript 4.5.2]

**Comment by thepandaguitar** [comment]
I am experiencing the same issue.

**Comment by HeyItsJs** [comment]
Facing the same issue with both left and right pins

**Comment by gejgalis-selector** [comment]
@makinggoodsoftware I'll try to reproduce in Plunkr/Stackblitz this week. In our project we have one usage of AgGrid where this issue occurs and the second one where everything works fine. I still don't know what is the reason.
I'll let you know if I find anything this week.

**Comment by makinggoodsoftware** [comment]
Hi @gejgalis-selector Thanks for that!

There is no rush, we will keep this open in case it takes you longer

Hoping we can help you with this!

**Comment by jameshelly** [comment]
I have also encountered this issue. 

I have investigated this for a few hours, my current theory is that it is tied to how the resizing method calculates things. 
It appears to happen when one of the pinned left columns are quite wide, more than 200px in our instance.

I'll see if I can get a chance to get an example up some where.

**Comment by daniellittledev** [comment]
I'm also trying to isolate this issue as it is effecting me, however I've so far been unable to reproduce it in a sample that I can share. What I can confirm is the @jameshelly seems to be onto something, when the grid is initially loaded the `ag-pinned-left-cols-container` is missing it's width. Resizing the column via the UI will also fix it.
![AGGrid Bug Pinning](https://user-images.githubusercontent.com/1761789/150908312-c9803752-c544-4628-80f4-53ca9bcf4962.gif)

**Comment by daniellittledev** [comment]
I was able to reproduce it!
https://codesandbox.io/s/ag-grid-reactui-error-pinning-fvd65?file=/src/App.tsx

**Comment by kiril-matev** [comment]
Hello @daniellittledev!

Thank you for sending this repro sample - much appreciated! 

Based on your description we've now reproduced this issue as a bug.

We have added this bug to our backlog and we are tracking it with the following reference and description:
**AG-6292 When using ReactUI=true with pinned columns, a second set state causes the pinned column cells to not display any cell values (because the pinned column has lost its width)**

We try to fix bugs from one release to the next, so this should be fixed in the next release or the one after if it was raised too close to the next release date.

You can see the next release date on the product pipeline page:
https://www.ag-grid.com/ag-grid-pipeline/

**Now that this issue is recorded in our backlog, we will soon close the GitHub issue because we will not update it when this bug is fixed in a future version of AG Grid. The best way to get an update on this item is to sign up for AG Grid new release notifications. Once a new AG Grid version is released, look for the item reference above (AG-XXXX) on the changelog page to see if it's resolved in that version:
https://www.ag-grid.com/ag-grid-changelog/**

Thanks again for bringing this up with us - we appreciate your time!

**Comment by abhijeetJaisw** [comment]
Hi! any fix for this yet?

**Comment by AG-Zoheil** [comment]
Hi @abhijeetJaisw,

Thank you for your comment.

As you can see this is quite an old issue from **more than two years ago**. The item in question **AG-6292** was released in a fix for version `27.0.0`.

The grid has naturally advanced a lot since then. If you are still having problems, can we please ask you to open a new issue with a reproducible live scenario - you can use one of the plunker examples from our [documentation](https://ag-grid.com/javascript-data-grid/) as a starting point.

This would help our team support in the most efficient way, and also increase visibility for the wider AG Grid community in case of contributions and comments.

Much appreciated!

Kind regards,
Zoheil
